codalab/model/bundle_model.py

```diff
@@ -221,9 +225,11 @@ def get_bundle_worker(self, uuid):
             row = conn.execute(
                 cl_worker_run.select().where(cl_worker_run.c.run_uuid == uuid)
             ).fetchone()
-            precondition(
-                row, 'Trying to find worker for bundle {} that is not running.'.format(uuid)
-            )
+
+            if len(row) == 0:
+                logger.info('Trying to find worker for bundle {} that is not running.'.format(uuid))
+                return None
```

Comment:
Can the caller deal with `None`?

Reply:
Yes, the caller handles `None` return result at https://github.com/codalab/codalab-worksheets/blob/master/codalab/server/bundle_manager.py#L266